sphinx-verilog-domain/verilogdomain.py

```diff
@@ -662,13 +662,3 @@ def resolve_xref(self, env, fromdocname, builder, typ, target, node, contnode):
         qualified_name_without_root = VerilogQualifiedIdentifier(obj.qualified_name[1:])
         tooltip = f"{obj.objtype} {qualified_name_without_root}" if obj.objtype else f"{qualified_name_without_root}"
         return make_refnode(builder, fromdocname, obj.docname, obj.linktarget, contnode, tooltip)
-
```

Comment:
Why are you removing this?

Reply:
This part has been moved to the [`__init__.py` file](https://github.com/SymbiFlow/sphinx-verilog-domain/pull/13/files#diff-bf04fac69205a9863fcfebcea57819a5R4) in the same fashion as it is done for [sphinxcontrib-verilog-diagrams](https://github.com/SymbiFlow/sphinxcontrib-verilog-diagrams/blob/master/sphinxcontrib_verilog_diagrams/__init__.py#L476) and i.e. [Breathe plugin](https://github.com/michaeljones/breathe/blob/master/breathe/__init__.py#L10).